ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/IUBundleContainer.java

```diff
@@ -684,6 +716,18 @@ public String serialize() {
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_ALL_PLATFORMS, Boolean.toString(getIncludeAllEnvironments()));
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_SOURCE, Boolean.toString(getIncludeSource()));
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_CONFIGURE_PHASE, Boolean.toString(getIncludeConfigurePhase()));
+		// Include references was only added quite late, defaults to true and
+		// most users will never edit it.
+		// As such, for stability, conciseness, and readability, we specifically
+		// don't serialize its default state.
+		boolean includeReferences = getIncludeReferences();
+		if (includeReferences) {
```

Comment:
```suggestion
		boolean includeReferences = getIncludeReferences();
		if (includeReferences) { // only serialize the a non-default value
```
The comment can be shorter here. The intend should be clear.